bbn-test-4/finality-providers/scripts/fp-changed.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -eu
+
+# USAGE:
+# ./fp-changed.sh
+
+# returns the finality provider modified files
+
+function fpChanged {
+  originUrl=$(git remote get-url origin)
+  currBranch=$(git rev-parse --abbrev-ref HEAD)
+  if [ "main" == "$currBranch" ] && [[ "$originUrl" =~ .*"babylonchain/networks.git".* ]]; then
+    echo "Merge on main"
```

Comment:
Do we need this echo?

Reply:
nop, can remove it

Reply:
https://github.com/babylonchain/networks/pull/193/commits/60cc002ab95c1225b4a2137fda24f3f262a05215